code/huffman/ViewModel/src/test/java/ru/unn/agile/huffman/viewmodel/HuffmanViewModelTest.java

```diff
@@ -0,0 +1,57 @@
+package ru.unn.agile.huffman.viewmodel;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class HuffmanViewModelTest {
+    private HuffmanViewModel viewModel;
+
+    @Before
+    public void setUp() {
+        viewModel = new HuffmanViewModel();
+    }
+
+    @After
+    public void tearDown() {
+        viewModel = null;
+    }
+
+    @Test
+    public void checkRefLabelTextByDefaultText() {
+        assertEquals(viewModel.getInput().get(), "Reference string");
```

Comment:
First argument is expected obj. Second - actual.
Could you swap here and below? Thx

Reply:
Done